src/main/java/ch/njol/skript/SkriptAddon.java

```diff
@@ -76,6 +77,20 @@ public SkriptAddon loadClasses(String basePackage, String... subPackages) throws
 		return this;
 	}
 
+	/**
+	 * Register a VariableStorage class for Skript to create if the user config value matches.
+	 * 
+	 * @param <T> A class to extend VariableStorage.
+	 * @param storage The class of the VariableStorage implementation.
+	 * @param names The names used in the config of Skript to select this VariableStorage.
+	 * @return This SkriptAddon for method chaining.
+	 * @throws SkriptAPIException if the operation was not successful because the storage class is already registered.
+	 */
+	public <T extends VariableStorage> SkriptAddon registerStorage(Class<T> storage, String... names) throws SkriptAPIException {
```

Comment:
Given that this class is now deprecated in favor of the `org.skriptlang` equivalent, I do not think there is a need to add this method. I also don't think there needs to be an equivalent on the main class. Pointing developers to `Variables#registerStorage` should be fine.